realm/realm-jni/src/io_realm_internal_LinkView.cpp

```diff
@@ -248,3 +250,15 @@ JNIEXPORT void JNICALL Java_io_realm_internal_LinkView_nativeRemoveTargetRow
         return lvr->remove_target_row( S(pos) );
     } CATCH_STD()
 }
+
+static void finalize_link_view(jlong ptr)
+{
+    TR_ENTER_PTR(ptr)
```

Comment:
Hmm, after checking, the `LinkView.nativeClose` is called by bulk insertion for speed. We need to keep that.